docs/organizations/overview.mdx

```diff
@@ -5,26 +5,140 @@ description: Learn about how to create and manage Clerk organizations and their
+
+To allow suggestions in your application, you must first enable the feature in the Clerk Dashboard. See the [Verified domains](/docs/organizations/verified-domains#enable-verified-domains) guide for more information.
+
+Once enabled for your application, when an organization admin is creating a verified domain in your application, they are able to enable suggestions for that domain.
+
+#### Membership requests
+
+Membership requests are requests from users who want to join an organization. A membership request is created when a user sees a suggestion to join an organization and clicks the **Request to join** button. Therefore, membership requests are only available for organizations that have the [**Verified domains** feature enabled](#verified-domains) and the [**Automatic suggestions** feature enabled in both the Dashboard and for the specific domain](#suggestions).
+
+## Active organization
+
+There are two types of accounts that a user can have in a Clerk application: a personal account and an organization account. A personal account is the default account type, and it is created when a user signs up for your application. An organization account is created when a user creates or joins an organization within your application.
+
+When a user is a member of an organization, they can switch between their personal account and an organization account. The organization account that a user is currently viewing is called the **active organization**. The active organization determines which organization-specific data the user can access and which roles and permissions they have within the organization.
+
+When a user signs in to your application, they are viewing their personal account and no organization is set as active. Even if they are a member of only one organization, they must explicitly switch to that organization to set it as active.
+
+The easiest way to allow users to switch between their personal account and their organization account is to use Clerk's [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) component. Once a user selects an organization, that organization is set as the active organization.
+
+## Organization invitations
+
+Organization invitations are a way to invite users to join an organization. When a user is invited to an organization, they will receive an email with a link to accept the invitation. Once they accept the invitation, they will be added as a member of the organization.
+
+By default, only admins can invite users to an organization.
+
+This feature requires that **Email address** is enabled as an [identifier](/docs/authentication/configuration/sign-up-sign-in-options#identifiers), as Clerk uses the user's email address to send the invitation. You can still disable **Email address** as an authentication option if you do not want users to be able to sign-in with their email address.
+
+To configure your application's **Email address** settings:
```

Comment:
We should have a doc or section of the docs dedicated to email stuff so we could link to it in situations like this. Feels kinda odd to have instructions for accessing email settings in the organizations overview. This can be bad because down the line if these instructions change, instead of having one central place to update, these steps may be in scattered places like this and end up forgotten.

For now, we'll just have to leave it, as consolidating all the email stuff into a concrete doc/guide/whatever is a whole other can of worms. I just wanted to point it out